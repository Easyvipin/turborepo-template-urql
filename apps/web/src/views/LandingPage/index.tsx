import HelloWorld from "@components/HelloWorld";
import React from "react";

interface ILandingPageProps {}

const LandingPage: React.FunctionComponent<ILandingPageProps> = (props) => {
  return (
    <div>
      <HelloWorld />
    </div>
  );
};

export default LandingPage;
