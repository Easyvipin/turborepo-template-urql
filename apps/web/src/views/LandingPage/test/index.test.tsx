import React from "react";
import { render } from "@testing-library/react";
import LandingPage from "../index";

test("renders LandingPage snapshot", () => {
  const { container } = render(<LandingPage />);
  expect(container).toMatchSnapshot();
});
