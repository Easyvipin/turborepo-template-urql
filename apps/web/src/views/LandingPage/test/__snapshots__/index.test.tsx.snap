// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LandingPage snapshot 1`] = `
<div>
  <div>
    <div>
      HelloWorld
    </div>
  </div>
</div>
`;
